sonar-php-plugin/src/main/java/org/sonar/plugins/php/phpunit/JUnitLogParserForPhpUnit.java

```diff
@@ -41,32 +35,13 @@
 public class JUnitLogParserForPhpUnit {
 
   public TestSuites parse(File report) {
-    try (InputStream inputStream = new FileInputStream(report)) {
-      final Object parsedObject = xstream().fromXML(inputStream);
-      return (TestSuites) parsedObject;
-    } catch (IOException | XStreamException e) {
+    try {
+      // TODO xstream.aliasSystemAttribute("fileName", "class");
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1135))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.php%3Aphp&branch=SONARPHP-758&issues=AWFIRjTHzrLehFnPs_1g&open=AWFIRjTHzrLehFnPs_1g)

[ISSUE_KEY]: AWFIRjTHzrLehFnPs_1g